ccd-definition/AuthorisationCaseField/CareSupervision/superuser.json

```diff
@@ -69,6 +76,20 @@
     "UserRole": "caseworker-publiclaw-superuser",
     "CRUD": "CRU"
   },
+  {
+    "LiveFrom": "01/01/2017",
+    "CaseTypeID": "CARE_SUPERVISION_EPO",
+    "CaseFieldID": "unlinkedHearing",
+    "UserRole": "caseworker-publiclaw-superuser",
+    "CRUD": "CRU"
```

Comment:
we haven't actually unlinked any hearing yet until they progress through the event, hearingToUnlink could be better but i'm not fussed if you would like to keep as is